### PR TITLE
Eruizgar91/w40 upstream merge

### PR DESCRIPTION
**Pull request by obasilakis**
W40 changes

#[4739](https://github.com/blockscout/blockscout/pull/4739) - Improve logs and inputs decoding
    #[4747](https://github.com/blockscout/blockscout/pull/4747) - Advanced CSV export
    #[4745](https://github.com/blockscout/blockscout/pull/4745) - Vyper contracts verification
    #[4699](https://github.com/blockscout/blockscout/pull/4699) - Address page facelifting
    #[4667](https://github.com/blockscout/blockscout/pull/4667) - Transaction Page: Add expand/collapse button for long contract method data
    #[4641](https://github.com/blockscout/blockscout/pull/4641), #[4733](https://github.com/blockscout/blockscout/pull/4733) - Improve Read Contract page logic
    #[4660](https://github.com/blockscout/blockscout/pull/4660) - Save Sourcify path instead of filename
    #[4656](https://github.com/blockscout/blockscout/pull/4656) - Open in Tenderly button
    #[4655](https://github.com/blockscout/blockscout/pull/4655), #[4676](https://github.com/blockscout/blockscout/pull/4676) - EIP-3091 support
    #[4621](https://github.com/blockscout/blockscout/pull/4621) - Add beacon contract address slot for proxy

### CODE REVIEW

**Review by eruizgar91 — APPROVED** [review]

File: apps/block_scout_web/lib/block_scout_web/templates/address/index.html.eex [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <%= render BlockScoutWeb.Advertisement.TextAdView, "index.html", conn: @conn %>
   <div class="card">
     <div class="card-body" data-async-load data-async-listing="<%= @current_path %>">
-      <h1 class="card-title list-title-description"><%= gettext "Addresses" %></h1>
+      <h1 class="card-title list-title-description"><%= gettext "Ether" %> <%= gettext "Addresses" %></h1>
```
Comment (rkachowski) [inline apps/block_scout_web/lib/block_scout_web/templates/address/index.html.eex:5]: "Ether" needs to be changed to "CELO"

File: apps/block_scout_web/lib/block_scout_web/templates/address_token/overview.html.eex [hunk 2]
```diff
@@ -0,0 +1,64 @@
+<% native_coin = gettext("ETH") %>
```
Comment (rkachowski) [inline apps/block_scout_web/lib/block_scout_web/templates/address_token/overview.html.eex:1]: ETH -> CELO

File: apps/block_scout_web/lib/block_scout_web/web_router.ex [hunk 3]
```diff
@@ -35,7 +35,13 @@ defmodule BlockScoutWeb.WebRouter do
       singleton: true
     )
 
-    resources "/blocks", BlockController, only: [:index, :show], param: "hash_or_number" do
+    resources "/block", BlockController, only: [:show], param: "hash_or_number" do
```
Comment (rkachowski) [inline apps/block_scout_web/lib/block_scout_web/web_router.ex:38]: nice!

File: cloudbuild.yaml [hunk 4]
```diff
@@ -1,5 +1,5 @@
 steps:
-  - name: "gcr.io/kaniko-project/executor:latest"
+  - name: "gcr.io/kaniko-project/executor:v1.6.0"
```
Comment (rkachowski) [inline cloudbuild.yaml:2]: what happened with the kaniko version?

File: docker/Dockerfile [hunk 5]
```diff
@@ -1,10 +1,21 @@
-FROM bitwalker/alpine-elixir-phoenix:1.11.4
+FROM bitwalker/alpine-elixir-phoenix:1.12
 
 RUN apk --no-cache --update add alpine-sdk gmp-dev automake libtool inotify-tools autoconf python3 file pango-dev jpeg-dev libjpeg-turbo-dev giflib-dev librsvg-dev
 
 ENV GLIBC_REPO=https://github.com/sgerrand/alpine-pkg-glibc
 ENV GLIBC_VERSION=2.30-r0
 
+RUN set -ex && \
```
Comment (rkachowski) [inline docker/Dockerfile:8]: This appears to be a duplicate and expanded version of the RUN command at line 19, probably we don't need the version on line 19 anymore

**Review by rkachowski — COMMENTED** [review]
Pretty cool, only some minor comments about copy changes (eth -> celo).

**Pull request merged**